WeatherApp/WeatherApp.Webpage/Services/WeatherService.cs

```diff
@@ -5,28 +5,56 @@
 using System.Net.Http;
 using System.Threading.Tasks;
 using WeatherApp.WebSite.Models;
+using WeatherApp.WebSite.Services.Interfaces;
 
 namespace WeatherApp.WebSite.Services
 {
-    public class WeatherForecastService : IWeatherForecastService
+    public class WeatherService : IWeatherService
     {
         readonly string _apiKey;
         readonly string _baseUrl;
 
         public HttpClient Client { get; set; }
 
-        public WeatherForecastService(IConfiguration configuration, HttpClient client)
+        public WeatherService(IConfiguration configuration, HttpClient client)
         {
             _apiKey = configuration["WeatherApp:ServiceApiKeys:Openweathermap"];

```

Comment:
Config key not following naming convention: `OpenWeatherMap`.

Reply:
Thanks for noticing, it's fixed now.